login.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+	session_start();
```

Comment:
This should be in the block where we determine that the users login was successful